client/src/js/components/bhCurrencySelect.js

```diff
@@ -0,0 +1,98 @@
+angular.module('bhima.components')
+.component('bhCurrencySelect', {
+  controller : bhCurrencySelect,
+  templateUrl : 'partials/templates/bhCurrencySelect.tmpl.html',
+  bindings : {
+    validationTrigger : '<',
+    currencyId : '=',
+    disableIds : '<',
+    onChange : '&'
+  }
+});
+
+bhCurrencySelect.$inject = [ '$scope', 'CurrencyService' ];
```

Comment:
@dedrickenc, could you please give me feedback on how to make this code better?  I do not know  a better way than this.  What suggestions do you have?